src/ert/plugins/hook_implementations/forward_model_steps.py

```diff
@@ -278,13 +281,17 @@ def __init__(self) -> None:
             default_mapping={"<NUM_CPU>": 1, "<OPTS>": "", "<VERSION>": "version"},
         )
 
-    def validate_pre_experiment(self, _: ForwardModelStepJSON) -> None:
+    def validate_pre_experiment(
+        self, _: ForwardModelStepJSON, env_vars: Dict[str, str]
```

Comment:
It's a bit unfortunate to break the signature of this function as it is public and used by dependents. I think it would be better to make env_vars part of `ForwardModelStepJSON` and be a non-required field. That way we do not break any dependents.

Reply:
Absolutely agree, this has already been done in #9108